pkg/distro/fedora/package_sets.go

```diff
@@ -1,10 +1,18 @@
 package fedora
 
 import (
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/distro"
 	"github.com/osbuild/images/pkg/distro/defs"
 	"github.com/osbuild/images/pkg/rpmmd"
 )
 
 func packageSetLoader(t *imageType) (rpmmd.PackageSet, error) {
 	return defs.PackageSet(t, "", VersionReplacements())
 }
+
+func imageConfig(d distribution, imageType string) *distro.ImageConfig {
+	// arch is currently not used in fedora
+	arch := ""
```

Comment:
This should probably still be wired up or there will be a surprise if we try to use it.